src/output-view.js

```diff
@@ -0,0 +1,107 @@
+import {
+    Transformime,
+    TextTransformer,
+    ImageTransformer,
+    HTMLTransformer
+} from "transformime";
+
+import {
+    consoleTextTransform,
+    markdownTransform,
+    LaTeXTransform,
+    PDFTransform
+} from "transformime-jupyter-transformers";
+
+/**
+ * Displays output area state.
+ */
+export class OutputView {
+    
+    /**
+     * Public constructor
+     * @param  {IOutputModel} model    output model that this view represents
+     * @param  {Document}     document handle to the Document instance that 
+     *                                 the output will be rendered in.
+     */
+    constructor(model, document) {
+        this.model = model;
+        Object.defineProperty(this, 'document', {get: () => document});
+        Object.defineProperty(this, 'el', {get: () => document.createElement('div')});
```

Comment:
For some reason, my comment didn't post. Weird.

Does this create an element every time `this.el` is called?